src/site/_filters/paginate.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Take array of paginated hrefs and select the pages to display.
+ * @param {string[]} pages An array of hrefs of all the pages.
+ * @param {string} current Href of current page.
+ * @return {object[]} An array of up to 10 items to display, including href and index.
+ */
+module.exports = function paginate(pages, current) {
```

Comment:
I'm not sure I understand what this filter is for. Can you explain a bit more why we need it?

Reply:
So it exists so that we do not display all the pages at the same time, just 10 pages at max 
(and it picks the 10 pages to show depending on the index of the page theyre currently on)